package.json

```diff
@@ -42,6 +42,10 @@
     "@angular/platform-browser": "2.0.0-rc.1",
     "@angular/platform-browser-dynamic": "2.0.0-rc.1",
     "@angular/router": "2.0.0-rc.1",
+    "@angular2-material/core": "^2.0.0-alpha.4",
+    "@angular2-material/sidenav": "^2.0.0-alpha.4",
+    "@angular2-material/button": "^2.0.0-alpha.4",
+    "@angular2-material/icon": "^2.0.0-alpha.4",
```

Comment:
the ^ allow npm to upgrade automatically. BTW, it seems we have to use 5-2 cause the previous build was pretty crappy.